app/views/govuk_publishing_components/components/_list.html.erb

```diff
@@ -1,12 +1,12 @@
 <%
-  id ||= nil
+  aria_label ||= nil
```

Comment:
Not blocking, just curious why you moved these lines around?

Reply:
So they're alphabetical - I find it makes it easier to find a specific variable. And alphabetical because it's simple and unambiguous.